docs/import/add_disaster.js

```diff
@@ -191,7 +237,13 @@ function withText(td, layer, property) {
  * @return {JQuery<HTMLTableDataCellElement>}
  */
 function withList(td, layer, property) {
-  return td.text(layer[property][0]);
+  // use a regex to replace all instances of ',' with a new line
+  const textarea = $(document.createElement('textarea'))
+                       .val(layer[property].toString().replace(/,/g, '\n'))
```

Comment:
I don't think modifying values on display is the way to go. If we want to split based on commas (we can, but I'm not clear we need to), I think we should do it on value extraction.

Reply:
Not totally sure what you mean by "modifying values on display" - is this better?